### PR TITLE
Create LIT7106Shinbone.xml

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
According to our discussion in https://github.com/BetaMasaheft/Documentation/issues/2628.

### CODE REVIEW

File: new/LIT7106Shinbone.xml [hunk 1]
```diff
@@ -0,0 +1,68 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7106Shinbone" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ጸሎት፡ በእንተ፡ አቁያጻት፡</title>
```
Comment (karljonaskarlsson) [inline new/LIT7106Shinbone.xml:8]: Is it better to give the "correct" አቍያጻት, ʾaqʷyāṣāt, in the title? I guess that is what people would be searching for if they are looking for parallels in other works. Or is it better to keep the spelling of the manuscript? I'm not sure :)

File: new/LIT7106Shinbone.xml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Magic prayer using <foreign xml:lang="gez-tr">ʾasmāt</foreign> to protect the shinbone 
+                    (<foreign xml:lang="gez-tr">ʾaquyāṣāt</foreign>) and against a witch doctor who killed a person before his time.</p>
```
Comment (karljonaskarlsson) [inline new/LIT7106Shinbone.xml:39]: Maybe it is better to understand the Geez more generally? "…and against witch doctors who kill persons before their time"

**Review by karljonaskarlsson — APPROVED** [review]

File: new/LIT7106Shinbone.xml [hunk 3]
```diff
@@ -0,0 +1,68 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7106Shinbone" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ጸሎት፡ በእንተ፡ አቁያጻት፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Ṣalot baʾǝnta ʾaquyāṣāt</title>
```
Comment (eu-genia) [inline new/LIT7106Shinbone.xml:9]: qʷ

File: new/LIT7106Shinbone.xml (hunk 2 above)
Comment (eu-genia) [reply new/LIT7106Shinbone.xml:39]: qʷ

File: new/LIT7106Shinbone.xml (hunk 1 above)
Comment (CarstenHoffmannMarburg) [reply new/LIT7106Shinbone.xml:8]: Yes, it is better to spell it according to Leslau in the titles.

**Pull request merged**